radicle-cli/src/commands/node/control.rs

```diff
@@ -0,0 +1,54 @@
+use radicle::node::{Address, Handle as _, NodeId};
+use radicle::Node;
+
+use crate::terminal as term;
+
+pub fn start() -> anyhow::Result<()> {
+    todo!()
+}
+
+pub fn stop(node: Node) -> anyhow::Result<()> {
+    let spinner = term::spinner("Stopping the node...");
+    if let Err(err) = node.shutdown() {
+        spinner.finish();
+        term::warning(&format!("Error occurred while shutting down node: {err}"));
```

Comment:
You can use `spinner.error()` or `spinner.warning()` here, instead of the `term::warning`

Reply:
Ah thanks! I missed these methods :)